### DISCUSSION TITLE
trying to: <apt install gnome-shell ubuntu-gnome-desktop error> too many packages that are not installable.

### DISCUSSION THREAD

**Discussion by ifishtoo** [johang/sd-card-images#146] [Q&A]
Got 24.04 installed and running from sd card on the bpi-r2 pro. I have internet. The resizepart 2 %100 worked out fine after I stumbled around for a while. 

This is my first time installing Ubuntu on an arm system. I may have missed some important reading somewhere. 
I have been using linux for decades. I am probably missing something somewhat obvious. I am rather excited about this repository because I hope to run android in a container so I can setup tinycam as a server. I will need openwrt in another container as well as tailscale, vnc-server and a few other utilities to run on ubuntu. 22.04 might be more mature for my application but the latest kernel opens up possibilities regarding optional network adapters.

apt install ubuntu-gnome-desktop
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 aptdaemon : Depends: policykit-1 but it is not installable
 dconf-service : Depends: default-dbus-session-bus or
                          dbus-session-bus
 gdm3 : Depends: default-logind or
                 logind
        Depends: gnome-session-bin (>= 3.37.0) but it is not installable
        Depends: polkitd but it is not installable
 gnome-settings-daemon : Depends: default-logind or
                                  logind
                         Recommends: pipewire-audio but it is not installable
                         Recommends: pkexec but it is not installable
 nautilus : Depends: xdg-desktop-portal-gnome but it is not installable or
                     xdg-desktop-portal-backend
 packagekit : Depends: polkitd but it is not installable
 tracker : Depends: default-dbus-session-bus or
                    dbus-session-bus
 ubuntu-desktop : Depends: gnome-control-center but it is not installable
                  Depends: ubuntu-session but it is not installable
                  Depends: wireplumber but it is not installable
                  Recommends: brltty but it is not installable
                  Recommends: file-roller but it is not installable
                  Recommends: firefox but it is not installable
                  Recommends: gamemode but it is not installable
                  Recommends: gnome-initial-setup but it is not installable
                  Recommends: gnome-keyring but it is not installable
                  Recommends: gnome-power-manager but it is not installable
                  Recommends: gnome-remote-desktop but it is not installable
                  Recommends: gnome-snapshot but it is not installable
                  Recommends: gnome-terminal but it is not installable
                  Recommends: libpam-fprintd but it is not installable
                  Recommends: network-manager but it is not installable
                  Recommends: network-manager-config-connectivity-ubuntu but it is not installable
                  Recommends: network-manager-openvpn-gnome but it is not installable
                  Recommends: network-manager-pptp-gnome but it is not installable
                  Recommends: remmina but it is not installable
                  Recommends: seahorse but it is not installable
                  Recommends: shotwell but it is not installable
                  Recommends: simple-scan but it is not installable
                  Recommends: snapd but it is not installable
                  Recommends: systemd-oomd but it is not installable
                  Recommends: thunderbird but it is not installable
                  Recommends: xdg-desktop-portal-gnome but it is not installable
 ubuntu-desktop-minimal : Depends: gnome-control-center but it is not installable
                          Depends: ubuntu-session but it is not installable
                          Depends: wireplumber but it is not installable
                          Recommends: brltty but it is not installable
                          Recommends: firefox but it is not installable
                          Recommends: gamemode but it is not installable
                          Recommends: gnome-initial-setup but it is not installable
                          Recommends: gnome-keyring but it is not installable
                          Recommends: gnome-power-manager but it is not installable
                          Recommends: gnome-remote-desktop but it is not installable
                          Recommends: gnome-terminal but it is not installable
                          Recommends: libpam-fprintd but it is not installable
                          Recommends: network-manager but it is not installable
                          Recommends: network-manager-config-connectivity-ubuntu but it is not installable
                          Recommends: network-manager-openvpn-gnome but it is not installable
                          Recommends: network-manager-pptp-gnome but it is not installable
                          Recommends: seahorse but it is not installable
                          Recommends: snapd but it is not installable
                          Recommends: systemd-oomd but it is not installable
                          Recommends: xdg-desktop-portal-gnome but it is not installable
 ubuntu-gnome-desktop : Depends: gnome-session but it is not installable
 update-manager : Depends: pkexec but it is not installable
                  Depends: polkitd but it is not installable

**Comment by johang** [comment]
Maybe you're missing some apt repositories or forgot to do `apt update`.
